package.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "geospatialdraw",
   "version": "0.5.7",
+  "private": true,
```

Comment:
Why do we want this to be private?

Reply:
Edit: just saw your slack message and that makes sense
`This will help prevent people from accidentally publishing the src in the future, since private is set in the package json.`